src/hooks/useStats.ts

```diff
@@ -77,13 +77,14 @@ export function getStats(
 }
 
 export function useAllSpeciesStatsForRegion(
-  region: Region,
-  stats: StatsBySpecies
+  region: Region | null,
+  stats: StatsBySpecies | null
 ) {
   return useMemo(() => {
+    if (!stats) return null
     return Object.fromEntries(
       Object.entries(stats).map(([spc, spcStats]) => {
-        const id = region.GID_1 || region.GID_0
+        const id = region ? region.GID_1 || region.GID_0 : 'global'
```

Comment:
Instead of checking for null here, I think 'global' should be set as a Region and come in from useRegionStats